lib/storage/providers/SQLiteStorage.js

```diff
@@ -88,10 +88,12 @@ const provider = {
              ON CONFLICT DO UPDATE
              SET valueJSON = JSON_PATCH(valueJSON, JSON(:value));
         `;
+
         const queryArguments = _.map(pairs, (pair) => {
-            const value = JSON.stringify(pair[1]);
-            return [pair[0], value];
+            const value = pair[1];
```

Comment:
Seems like this variable really isn't necessary then, is it?

Reply:
+1